gulpfile.js/build.js

```diff
@@ -355,24 +387,63 @@ function buildPages(done) {
         .pipe(gulp.dest(`${project.paths.PAGES_DEST}/shared`));
     },
     // eslint-disable-next-line prefer-arrow-callback
-    async function publishPages() {
-      if (!config.options.locales.includes(config.getDefaultLocale())) {
+    async function copyBuildFiles(done) {
+      if (!config.options?.locales?.includes(config.getDefaultLocale())) {
         console.log(
           'Skipping page publishing. Default language is not build, only:',
           config.options.locales
         );
-        return;
+        return done();
       }
 
+      await copyFile(
```

Comment:
would it make sense to parallelise copying the files?

Reply:
since it's two files each, wasnt sure if it made sense to do it more than this. but will update to do that